index.html

```diff
@@ -0,0 +1,1312 @@
+            <span class="bolder">
+              <a href="#" rel="noopener" class="no-decor color_b">
+                Scopio Authentic Stock Photography: Lifetime Subscription
+              </a>
+              <br />
+              <span class="color-gray">
+                <span class="dec sml">$3480</span> $29</span
+              >
+            </span>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="fot1">
+        <div class="d-flex just_cent col_gap">
+          <div class="fot-col d-flex just_cent">
+            <a href="#" rel="noopener" class="align_cent">
+              <i class="fab fa-facebook-square color-w"></i>
+            </a>
+          </div>
+          <div class="fot-col d-flex just_cent">
+            <a href="#" rel="noopener" class="align_cent">
+              <i class="fab fa-twitter color-w"></i>
+            </a>
+          </div>
+          <div class="fot-col d-flex just_cent">
+            <a href="#" rel="noopener" class="align_cent">
+              <i class="fab fa-youtube color-w"></i>
+            </a>
+          </div>
+          <div class="fot-col d-flex just_cent">
+            <a href="#" rel="noopener" class="align_cent">
+              <i class="fab fa-instagram color-w"></i>
+            </a>
+          </div>
+          <div class="fot-col d-flex just_cent">
+            <a href="#" rel="noopener" class="align_cent">
+              <i class="fab fa-pinterest color-w"></i>
+            </a>
+          </div>
+          <div class="fot-col d-flex just_cent">
+            <a href="#" rel="noopener" class="align_cent">
+              <i class="fas fa-rss color-w"></i>
+            </a>
+          </div>
+          <div class="fot-col d-flex just_cent">
+            <a href="#" rel="noopener" class="align_cent">
+              <i class="fas fa-envelope color-w"></i>
+            </a>
+          </div>
+        </div>
+
+        <div class="d-flex just_cent col_gap fot_hov marg_top_25">
+          <a href="#" rel="noopener" class="color-gray no-decor fot_hov_l"> EVENTS</a>
+          <a href="#" rel="noopener" class="color-gray no-decor fot_hov_l"> ABOUT</a>
+          <a href="#" rel="noopener" class="color-gray no-decor fot_hov_l"> ADVERTISE </a>
+          <a href="#" rel="noopener" class="color-gray no-decor fot_hov_l"> JOBS </a>
+          <a href="#" rel="noopener" class="color-gray no-decor fot_hov_l"> CONTACTS </a>
+        </div>
+      </div>
+      <div class="fot2 pad-10 d-flex just_cent color-w">
+        <a href="#" rel="noopener color-w">
+          <img class="sml_logo" src="images/TNW.svg" alt="Top logo" />
+        </a>
+
+        <span class="color-gray">
+          © 2006–2020 The Next Web B.V. Made with heart in Amsterdam. Powered by
+        </span>
+        <a href="#" rel="noopener" class="bolder color-w no-decor">
+          &nbsp; STACKPATH</a
+        >
+      </div>
+    </footer>
+  </body>
+</html>
```

Comment:
- [ ] On Mobile screens, make the texts appear on 3 lines 

| Yours | Original |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/11758151/92292754-b9939f80-ef16-11ea-9b69-3be21dceee87.png) | ![image](https://user-images.githubusercontent.com/11758151/92292771-c87a5200-ef16-11ea-886b-6b2d244fa3ec.png) |